src-electron/db/zap-schema.sql

```diff
@@ -808,6 +808,7 @@ CREATE TABLE IF NOT EXISTS "ENDPOINT" (
   "PROFILE" integer,
   "ENDPOINT_IDENTIFIER" integer,
   "NETWORK_IDENTIFIER" integer,
+  "PARENT" integer,
   foreign key (SESSION_REF) references SESSION(SESSION_ID) on delete cascade,
   foreign key (ENDPOINT_TYPE_REF) references ENDPOINT_TYPE(ENDPOINT_TYPE_ID) on delete
   set NULL
```

Comment:
There should be a foreign key relationship here, saying that `PARENT_ENDPOINT_REF` needs to point to an existing `ENDPOINT_ID`.